rdr_service/dao/consent_dao.py

```diff
@@ -92,3 +92,15 @@ def get_files_ready_to_sync(self, org_names=None, session: Session = None) -> Co
                 return self._get_ready_to_sync_with_session(session=dao_session, org_names=org_names)
         else:
             return self._get_ready_to_sync_with_session(session=session, org_names=org_names)
+
+    @classmethod
```

Comment:
Is there a reason to utilize the `@classmethod` on these methods and passing session as an arg instead of using the context manager for setting the session from self?

Reply:
For how this method is being used currently, no not really. But I've been finding that the DAOs will create a session connected to the local database even when the rest of the environment is targeting an environment (like Prod), so I've been getting in the habit of passing a session in to where it would be used rather than creating one from scratch.